kubevirtci/kubevirtci.sh

```diff
@@ -0,0 +1,58 @@
+#!/bin/bash
+#
+# Copyright 2022 Red Hat, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+set -ex
+
+_action=$1
+_pwd=$(pwd)
+
+function kubevirtci::registry() {
+  echo "localhost:$("./cluster-up/cli.sh" ports registry 2>/dev/null)"/kubevirt/ssp-operator
```

Comment:
`echo "localhost:$(./cluster-up/cli.sh ports registry 2>/dev/null)/kubevirt/ssp-operator"`

Does it work like this too? It looks a bit more readable like this.

Reply:
ACK I'll clean this up.